phpstan.neon

```diff
@@ -38,3 +38,4 @@ parameters:
         - */app/Helpers/ComposerScripts.php
         - */app/Http/Controllers/DAV/Backend/CardDAV/AddressBookHome.php
         - */app/Providers/BroadcastServiceProvider.php
+        - */app/Services/Account/Settings/ExportAccount.php
```

Comment:
This is bad!

Reply:
Yeah but try removing it and see how it triggers a really, really stupid error as it doesn't understand the context.